### PR TITLE
[doc] fix doc describes an error about Asynchronous Compaction

### LINKED ISSUE #4375: [Bug] The documentation describes an error about Asynchronous Compaction
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/paimon/issues) and found nothing similar.

### Paimon version

1.0-SNAPSHOT

### Compute Engine

none

### Minimal reproduce step

none

### What doesn't meet your expectations?

The website describes it as follows：
        num-sorted-run.stop-trigger = 2147483647
       sort-spill-threshold = 10
       lookup-wait = false
       
The correct description would be as follows:
        num-sorted-run.stop-trigger = 10
       sort-spill-threshold = 2147483647
       lookup-wait = false

### Anything else?

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by herefree**
### Purpose

Linked issue: close #4375

### CODE REVIEW

File: docs/content/primary-key-table/compaction.md [hunk 1]
```diff
@@ -57,8 +57,8 @@ expect a mode to have maximum writing throughput, the compaction can be done slo
 You can use the following strategies for your table:
 
 ```shell
-num-sorted-run.stop-trigger = 2147483647
-sort-spill-threshold = 10
+num-sorted-run.stop-trigger = 10
```
Comment (wwj6591812) [inline docs/content/primary-key-table/compaction.md:60]: If apply this config, the commit will be block by compaction when the number of sorted-run is 10.
Then writing throughput will decline.

Comment (wwj6591812) [inline docs/content/primary-key-table/compaction.md:60]: I think:
If apply this config, the commit will be block by compaction when the number of sorted-run is 10.
Then writing throughput will decline.
May I ask why you made this modification？

File: docs/content/primary-key-table/compaction.md [hunk 2]
```diff
@@ -57,8 +57,8 @@ expect a mode to have maximum writing throughput, the compaction can be done slo
...
+sort-spill-threshold = 2147483647
```
Comment (wwj6591812) [inline docs/content/primary-key-table/compaction.md:61]: I think:
If apply this config, no sorted-run will be spill when compaction or mergeOnRead, maybe cause OOM.
May I ask why you made this modification？

Comment (herefree) [reply docs/content/primary-key-table/compaction.md:61]: Sorry,I made a mistake.I'll close it.

**Pull request closed without merging**